app/instances/auth.py

```diff
@@ -0,0 +1,42 @@
+from flask import g, session
+from app.server import server
+from app.models.User import User
+from app.instances.db import r
+
+userid_skey = 'uid'
+skey_prefix = 'sid:'
+
+# In seconds
+session_time = 60 * 60 * 24
```

Comment:
Set to one day and everytime a request with a session cookie is made, that time is reset